saleor/graphql/schema.graphql

```diff
@@ -5043,19 +5044,37 @@ enum WeightUnitsEnum {
 
 type Wishlist implements Node {
   id: ID!
-  createdAt: DateTime!
+  token: UUID!
   items(before: String, after: String, first: Int, last: Int, id: ID): WishlistItemCountableConnection!
 }
 
-type WishlistAddProductMutation {
+type WishlistAddVariants {
+  errors: [Error!]!
+  wishlist: Wishlist
+  wishlistErrors: [WishlistError!]!
+}
+
+type WishlistRemoveVariants {
+  errors: [Error!]!
+  wishlist: Wishlist
+  wishlistErrors: [WishlistError!]!
+}
+
+type WishlistUpdateVariants {
   errors: [Error!]!
-  wishlist: [WishlistItem]
+  wishlist: Wishlist
   wishlistErrors: [WishlistError!]!
 }
 
-type WishlistAddProductVariantMutation {
+type WishlistUpdateVariantsInput {
+  id: ID!
+  quantity: Int!
+}
+
+type wishlistMoveItemsToCart {
   errors: [Error!]!
-  wishlist: [WishlistItem]
+  wishlist: Wishlist
+  outOfStockItems: [ID!]
```

Comment:
@mateuszkula Should we return here IDs of variants that were not added or you prefer to get these objects instead? So `[ID!]` vs. `[ProductVariant!]`?

Reply:
Lets use ProductVariants -  its more flexible on frontend to get all product info.